dbt/adapters/databricks/relation_configs/incremental.py

```diff
@@ -5,10 +5,11 @@
 from dbt.adapters.databricks.relation_configs.base import DatabricksRelationChangeSet
 from dbt.adapters.databricks.relation_configs.base import DatabricksRelationConfigBase
 from dbt.adapters.databricks.relation_configs.tags import TagsProcessor
+from dbt.adapters.databricks.relation_configs.tblproperties import TblPropertiesProcessor
 
 
 class IncrementalTableConfig(DatabricksRelationConfigBase):
-    config_components = [TagsProcessor]
+    config_components = [TagsProcessor, TblPropertiesProcessor]
```

Comment:
Previously we only handled Tags this way (i.e. this is a new approach to updating the metadata of incremental tables).  This structure checks for each configured component to see if there are any metadata changes.

Reply:
Shouldn't we also fix the update of the table comment?

Reply:
I need to double check, but I think the PR that I merged from @henlue yesterday could fix the incremental table comment issue.  If not, I'll send another PR to address that.

Reply:
Yes the incremental table comments were fixed as part of my PR. However, they are updated on every incremental run, whether or not they are already present in databricks.

Reply:
I think that is fine.  We check for diffs on column comments because we can take one action to save us from many (updating every column).  For tblproperties, we could similarly update without checking, but if I ever do move to add deletes (would kind of require Databricks to stop writing tblproperties internally), it will be good to have the checking logic in place.